### DISCUSSION TITLE
Harman SDK for iOS development tutorial please

### DISCUSSION THREAD

**Discussion by kurtongicognite** [airsdk/Adobe-Runtime-Support#1928] [Q&A]
Hello world, would like to ask any links or source (pdf, docs)

for iOS development using Harman SDK? Thank you.

**Comment by ajwfrost** [comment]
Are you already familiar with AIR and ActionScript? i.e. you just need support in getting set up for distributing your AIR apps on iOS?
Or is this from a completely new perspective?

Plus - do you have a mac development machine, or just Windows? as there are some things that are a bit easier to do via mac (see e.g. #1863)

thanks
